File: JAVASCRIPT/class-4[]/one.js
let employee=[
    {"eid":101,"ename":"srinithi","esal":45000.56},
    {"eid":102,"ename":"kokila","esal":56000.56},
    {"eid":103,"ename":"sasi","esal":55000.56},
    {"eid":104,"ename":"karthik","esal":85000.56}
]
//print all employees names:
console.log(employee[0].ename)
console.log(employee[1].ename)
console.log(employee[2].ename)
console.log(employee[3].ename)
//using "FOR OF" we can available to print names,id
for(emp of employee){
    console.log(emp)
    console.log(emp.eid)
    console.log(emp.esal)
    console.log("name of emp is:",emp.ename)
}

File: PYTHON-II/class-3[JSON FIle Handling]/Dumps&Loads/one.py
# loads---json===>python
import json
emp_str='''{"eid":101,"ename":"srinithi","avail":true},
{"eid":102,"ename":"deepa","avail":true},
{"eid":103,"ename":"karthik","avail":true},
{"eid":103,"ename":"subash","avail":true},
'''
# print(emp_str)
# print(type(emp_str))
# {"eid":101,"ename":"srinithi","avail":true},
# {"eid":102,"ename":"deepa","avail":true},
# {"eid":103,"ename":"karthik","avail":true},
# {"eid":103,"ename":"subash","avail":true},
# <class 'str'>

employees=json.loads(emp_str)
print(employees)
print(type(employees))